documentation/docs/scenarios/xsa-deploy/Readme.md

```diff
@@ -0,0 +1,96 @@
+# Build and Deploy SAPUI5 or SAP Fiori Applications on SAP Cloud Platform with Jenkins
+
+Build an application based on SAPUI5 or SAP Fiori with Jenkins and deploy the build result into an SAP Cloud Platform account in the Neo environment.
+
+## Prerequisites
+
```

Comment:
@1084565 @SarahNoack  I guess this comment refers to my `TODO` in the code :

> do we have a general description how to setup docker.

Is such a description available so that we can refer to it?

Reply:
We should not describe the docker setup. But maybe I'm missing some aspects.
How would it be different from https://docs.docker.com/get-started/ ?

Reply:
It is not about describing the docker setup here, that would be clearly the wrong place. But I'm looking for a page describing the docker setup so that I can refer to that from here. Above ypu proivded a like to an external page, that we can use, of course.